tests/custom_hypothesis_support.py

```diff
@@ -51,6 +51,11 @@
 binary_bool_operator = hs.sampled_from(['and', 'or'])
 unary_bool_operator = hs.sampled_from(['not'])
 
+# Strategies for generating builtin type names
+annotation_types = [bool, bytearray, bytes, complex, dict, enumerate
```

Comment:
The name of this variable is misleading - this is a collection of built-in types, independent of how you are using them (for annotations).